dts/bindings/mipi-dsi/nxp,imx-mipi-dsi.yaml

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright (c) 2022, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+description: NXP MCUX MIPI DSI
+
+compatible: "nxp,imx-mipi-dsi"
+
+include: mipi-dsi-host.yaml
+
+properties:
+    interrupts:
+      required: true
+
+    nxp,lcdif:
+      type: phandle
+      required: true
+      description:
+        Instance of the LCDIF peripheral.
+
+    dpi-color-coding:
+      type: string
+      enum:
+        - "16-bit config 1"
+        - "16-bit config 2"
+        - "16-bit config 3"
+        - "18-bit config 1"
+        - "18-bit config 2"
+        - "24-bit"
+      description:
+        MIPI DPI interface color coding.
```

Comment:
can "config N" meaning be described?

Reply:
This is what I found documented in our reference manual:
`Sets the distribution of RGB bits within the 24-bit d bus, as specified by the DPI specification.`
I will add this to the property description.